Account/FxAClient10.swift

```diff
@@ -237,36 +237,36 @@ public class FxAClient10 {
         mutableURLRequest.HTTPMethod = Method.POST.rawValue
 
         mutableURLRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
-        mutableURLRequest.HTTPBody = JSON(parameters).toString(pretty: false).utf8EncodedData
+        mutableURLRequest.HTTPBody = JSON(parameters).toString(false).utf8EncodedData
 
-        let request = alamofire.request(mutableURLRequest)
+        _ = alamofire.request(mutableURLRequest)
```

Comment:
Don't need to assign this to anything - especially if we're ignoring it. We can remove the _ = part